src/main/java/transactionexecutor/Operation.java

```diff
@@ -0,0 +1,9 @@
+package transactionexecutor;
+
+import fruittransaction.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface Operation {
+    Map<String, Integer> startOperation(List<FruitTransaction> products);
+}
```

Comment:
```suggestion
public interface TransactionHandler {
    int handle(FruitTransaction transaction);
}
```

this class should be responsible only for one transaction by calculating the next fruit quantity and returning it so executor can update storage